Sources/Datadog/RUM/DataModels/RUMDataModels.swift

```diff
@@ -300,6 +300,30 @@ public struct RUMActionEvent: RUMDataModel {
         }
     }
 
+    /// Display properties
+    public struct RUMDisplay: Codable {
```

Comment:
public API convention/ ([“avoid platform self referencing”](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/3027638506/Public+API+Convention+in+iOS+SDK+2.x#Rule-7%3A-Use-of-standard-Datadog-and-RUM-terminology)) Because now we nest this model, there is no more need to prefix it with `RUM*`. We add this prefix as part of RUM code decoration:
```swift
if fixedName == "Display" {
   fixedName = "RUMDisplay"
}
```
Otherwise we're self-referencing the product in public API:
```diff
- RUMViewEvent.RUMDisplay
- RUMActionEvent.RUMDisplay
...
+ RUMViewEvent.Display
+ RUMActionEvent.Display
```